spec/features/person_creation_spec.rb

```diff
@@ -13,36 +13,31 @@
     cert = create(:department, shortname: "CERT")
   end
 
-  scenario "from the everybody people page" do
-    visit everybody_people_path
-    click_link "People"
-    click_link "New Person"
+  scenario "from the people page a.k.a all active", js: true do
+    visit people_path
+    find('.add-btn').click
     fill_in "First Name", with: "John"
     fill_in "Last Name", with: "Doe"
     fill_in "Date of birth", with: "1990-05-13"
     select "Active", from: "person_status" #"Status" also matches Channel status
+    page.execute_script "window.scrollBy(0,1500)"
```

Comment:
Didn't know about execute_script, thanks...

Reply:
@kgf @ThomasCarlson Some of these forms are very large and have dynamic heights due to JS changing fields and content. This forces the page to scroll to bring the button in view. The arguments change depending on the height you want.